bundle/config/validate/unique_resource_keys.go

```diff
@@ -66,10 +65,7 @@ func (m *uniqueResourceKeys) Apply(ctx context.Context, b *bundle.Bundle) diag.D
 				}
 			}
 
-			// dyn.Path under the hood is a slice. The code that walks the configuration
-			// tree uses the same underlying slice to track the path as it walks
-			// the tree. So, we need to clone it here.
-			m.paths = append(m.paths, slices.Clone(p))
+			m.paths = append(m.paths, p)
```

Comment:
@andrewnester I recall you had a PR where you needed to make a similar change, but don't recall which one.

Was there another call site where we did this?

Reply:
@pietern it's in `expand_glob_references` https://github.com/databricks/cli/pull/1664#discussion_r1709518819

we have the same in expand_glob file

Reply:
Thanks, updated both.